packages/remix-react/package.json

```diff
@@ -16,8 +16,8 @@
   "typings": "dist/index.d.ts",
   "module": "dist/esm/index.js",
   "dependencies": {
-    "@remix-run/router": "1.0.4",
-    "react-router-dom": "6.4.4"
+    "@remix-run/router": "1.0.5",
```

Comment:
This was out of sync with the version used in `@remix-run/server-runtime` which caused issues during integration tests.  I don't _think_ this would be an actual problem in apps since server-runtime had it's own copy nested in a local `node_modules`, however we do some tricky stuff with our integration tests builds and I think everything reads from the global repo `node_modules` so it was picking up on the wrong version from `@remix-run/react`. 

Generally speaking we'll upgrade these in lock step going forward so I expect this is largely a non-issue - this mismatch is just a function of the PR to update to `react-router-dom@6.4.4` and the Remix `1.8.2` hotfix release that bumped to `@remix-run/router@1.0.5` merging side by side.